aiida/cmdline/commands/cmd_node.py

```diff
@@ -418,29 +418,37 @@ def verdi_graph():
     default='dot'
 )
 @click.option('-f', '--output-format', help="The output format used for rendering ('pdf', 'png', etc.).", default='pdf')
+@click.option(
+    '--target-cls', help='Only show nodes of target node class specified by class name', type=click.STRING, default=''
+)
```

Comment:
> Perhaps it would be better to use something along the lines of highlight_class or something similar.

Yes, `target-cls` seems a little ambiguous. `highlight_class` is better, will change to it. But I think highlight-class is to long as a cmd option, so I add `-c` (short for 'class') for this.

> it would be better to use None as the default for non-provided inputs.

I keep it a empty string here to make the variable assignment simple, otherwise `getattr(orm, target_cls, None)` will complain about the type of `target_cls`. 
The simple assignment code `getattr(orm, highlight_class, None)` for highlight_class will be:

    if highlight_class:
        highlight_class = getattr(orm, highlight_class)
    else:
        highlight_class = None

If the specification in the code base is using None for the situation, I'll adopt your suggestion. Please let me know about your preference.

> Would there be a way to add into the help string information so the user knows what is the string that needs to be provided here? 

I assume all builtin data class can be import by `orm.<className>` for example, `orm.FolderData` is equivalent to `orm.data.folder.FolderData`. Is that true for all the builtin data class and will this alias of data type remains invariable in future API?

Reply:
If fact, the argument `highlight_class` here can be a tuple of data type class. But I don't know how to implement it in cmdline interface. Maybe it can be realized by using `multiple=True` option of `click`. Are there other places in the code base `multiple=True` be used?

Reply:
Uhm, I see what you mean. Notice however that if `highlight_class` is None by default, then you don't really need the `else` clause of that conditional, right? Just the following should be enough:

```
    if highlight_class:
        highlight_class = getattr(orm, highlight_class, None)
```

Regarding the `multiple=True` option, from what I could find this is to be able to pass multiple times the flag (so `--highlight_class class1 --highlight_class class2`), does it also allow you to pass a single list? (`--highlight_class=[class1, class2]`). This indeed would be useful versatility.